content/influxdb/v2.0/reference/rc0-upgrade-guide.md

```diff
@@ -0,0 +1,271 @@
+
+```sh
+mv ~/.influxdbv2 ~/.influxdbv2_old
+```
+
+(You can move it to wherever you'd like.)
+When we start the old instance again, we will tell it where your data files are located.
+<!-- we'll provide this location -->
+
+Start the latest InfluxDB version by running
+
+```sh
+influxd
+```
+
+Since the data folder has been moved, everything will be empty.
+<!-- You can check out http://localhost:8086 in your browser and see a setup page, but don't set it up yet. -->
+You can check out http://localhost:8086 in your browser and see a setup page, but don't go through the setup process yet.
+
+## 5. Start old InfluxDB beta instance
+
+You can now start your old influxdb instance and point it to your old data directory:
+
+```sh
+influxd_old --bolt-path ~/.influxdbv2_old/influxd.bolt --engine-path ~/.influxdbv2_old/engine
+```
+
+If you used a different location, update as appropriate.
+
+Double check that InfluxDB is working by going to your previous InfluxDB beta location (probably http://localhost:9999) and logging in.
+Everything should still be there.
+
+{{% note %}}
+If you run into a problem about a missing migration, you can manually edit your bolt file to remove it.
+You can use BoltBrowser to open and edit your old influxd.bolt file and manually remove the offending migration.
+
+{{% warn %}}
+Make a backup of your old influxd.bolt file before doing this, as manually editing your bolt file could cause you to lose all your data.
+{{% /warn %}}
+
+Highlight the record under the `migrationsv1` path and press **D**.
+{{% /note %}}
+
+At this point, you should have the new instance and old instance of InfluxDB running on the same machine.
+
+## 6. Configure configuration profiles for the InfluxDB CLI
+
+Next, set up your InfluxDB CLI to connect to your old and new instances.
+If you've used the CLI before, you can copy your existing config file to your new data directory:
+
+```sh
+cp ~/.influxdbv2_old/configs ~/.influxdbv2/configs
+```
+
+You might want to edit that file and rename your old config to be something to indicate that it's the old version, like `influx_old`.
+
+{{< keep-url >}}
+```toml
+[influx_old]
+  url = "http://localhost:9999"
+  token = "<YOUR TOKEN>"
+  org = "influxdata"
+  active = true
+```
+
+If you've never used the CLI before, you can create a new configuration profile using the `influx config` command.
+
+```sh
+influx config create \
+    --config-name influx_old \
+    --host-url http://localhost:9999 \
+    --org influxdata --token <OLD_TOKEN>
+```
+
+If you run `influx config ls` after this, you should see a profile for your old instance.
+
+```sh
+$ influx config ls
+Active  Name        URL                    Org
+*       influx_old  http://localhost:8086  InfluxData
+```
+
+Next, let's set up your new instance which will automatically create a configuration profile for you.
+You can use the same username and password as your old instance, or something completely new, it's up to you.
```

Comment:
Since the `influx setup` command will create a config profile named `default` automatically, you will want to make sure you don't already have a profile with that name before running the setup command.